tasks/main.yml

```diff
@@ -111,6 +111,8 @@
         cp /etc/postfix/main.cf
         /etc/postfix/main.cf.{{ postfix_backup_multiple |
         ternary("$(date -Iseconds)", "backup") }}
+      args:
+        executable: /bin/bash
```

Comment:
Why is this needed? Does Debian use some other executable?

Reply:
Actually, this task copies files, and we can use the `copy` module instead of using `shell`. There is an old #2 that nobody looked at for years, but it has some good code that would fix this. Can you use it here?

Reply:
I am about to merge #2, which will fix the issue with pipefail. This PR now lacks adding compatibility with Debian in meta/main.yml, like in https://github.com/linux-system-roles/ssh/blob/main/meta/main.yml#L18-L23

Reply:
Cool, that will also fix it :)

Shall I close this one then?